modules/installation-initializing.adoc

```diff
@@ -14,20 +14,32 @@ You can customize your installation of {product-title} on a compatible cloud.
 
 .Procedure
 
-
-. Obtain the `*./install-config.yaml*` file.
+. Obtain the `./install-config.yaml` file.
 .. Run the following command:
 +
+[source,bash]
 ----
-$ openshift-install init
+$ ./openshift-install create install-config --dir=<installation-directory> <1>
 ----
-.. At the prompts, provide the configuration details for your cloud.
-
-. If required, modify the `*./install-config.yaml*` file. In this file, you can
-provide more values.
-
-. Validate the contents of the `*./install-config.yaml*` file.
+<1> Specify the directory name to store the files that the installation program
+creates.
+.. At the prompts, provide the configuration details for your cloud:
+... Select an SSH key to use to access your cluster machines.
+... Select AWS as the platform to target.
+... Select the AWS region to deploy the cluster to.
+... Select the base domain for the Route53 service that you configured for your cluster.
+... Enter a descriptive name for your cluster.
+... Paste the pull secret that you obtained from link:https://try.openshift.com[the OpenShift start page].
+
+. Modify the `./install-config.yaml` file. You can find more information about
```

Comment:
At this point you have established a placeholder for the asset directory, so you can use `<installation-directory>/install-config.yaml`.  Or leave the directory implicit and use `install-config.yaml`, but `./` resolves to the current working directory, so readers might not recognize an implicit asset directory, look in their working directory, and not find the file.

Reply:
You're right - I changed several references to this file.